config/crossSections13p6TeV.yaml

```diff
@@ -0,0 +1,718 @@
+  unc: 29320.0
+QCD_PT-20to30_MuEnrichedPt5:
+  crossSec: 2679000.0
+  reference: XSDB QCD_PT-20to30_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:54
+  unc: 26580.0
+QCD_PT-30to50_EMEnriched:
+  crossSec: 6698000.0
+  reference: XSDB QCD_PT-30to50_EMEnriched_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-05-26 21:19:12
+  unc: 62870.0
+QCD_PT-30to50_MuEnrichedPt5:
+  crossSec: 1465000.0
+  reference: XSDB QCD_PT-30to50_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:41
+  unc: 14360.0
+QCD_PT-50to80_EMEnriched:
+  crossSec: 2113000.0
+  reference: XSDB QCD_PT-50to80_EMEnriched_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-05-26 21:19:12
+  unc: 19470.0
+QCD_PT-50to80_MuEnrichedPt5:
+  crossSec: 409500.0
+  reference: XSDB QCD_PT-50to80_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-05-26 21:19:13
+  unc: 4049.0
+QCD_PT-80to120_EMEnriched:
+  crossSec: 394100.0
+  reference: XSDB QCD_PT-80to120_EMEnriched_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:48
+  unc: 3654.0
+QCD_PT-80to120_MuEnrichedPt5:
+  crossSec: 96200.0
+  reference: XSDB QCD_PT-80to120_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:49
+  unc: 916.9
+QCD_PT-120to170_EMEnriched:
+  crossSec: '0.0'
+  reference: fill me!
+  unc: ''
+QCD_PT-120to170_MuEnrichedPt5:
+  crossSec: 22980.0
+  reference: XSDB QCD_PT-120to170_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:51
+  unc: 215.1
+QCD_PT-170to300_EMEnriched:
+  crossSec: 17920.0
+  reference: XSDB QCD_PT-170to300_EMEnriched_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:33
+  unc: 164.5
+QCD_PT-170to300_MuEnrichedPt5:
+  crossSec: 7763.0
+  reference: XSDB QCD_PT-170to300_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2024-04-03 19:47:43
+  unc: 23.67
+QCD_PT-300_EMEnriched:
+  crossSec: 1231.0
+  reference: XSDB QCD_PT-300_EMEnriched_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:37
+  unc: 11.28
+QCD_PT-300to470_MuEnrichedPt5:
+  crossSec: 699.1
+  reference: XSDB QCD_PT-300to470_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:38
+  unc: 6.639
+QCD_PT-300toInf_EMEnriched:
+  crossSec: 1224.0
+  reference: XSDB QCD_PT-300toInf_EMEnriched_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:39
+  unc: 11.46
+QCD_PT-470to600_MuEnrichedPt5:
+  crossSec: 68.24
+  reference: XSDB QCD_PT-470to600_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2024-04-03 19:47:45
+  unc: 0.2049
+QCD_PT-600to800_MuEnrichedPt5:
+  crossSec: 21.37
+  reference: XSDB QCD_PT-600to800_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-05-26 21:19:13
+  unc: 0.199
+QCD_PT-800to1000_MuEnrichedPt5:
+  crossSec: 3.913
+  reference: XSDB QCD_PT-800to1000_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2023-03-26 13:58:47
+  unc: 0.03526
+QCD_PT-1000_MuEnrichedPt5:
+  crossSec: 1.323
+  reference: XSDB QCD_PT-1000_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8 modifiedOn 2024-04-03 19:47:38
+  unc: 0.003921
```

Comment:
separate `EMEnriched` and `MuEnrichedPt5` into different "sections"

Reply:
Fixed in later commit